control-plane/pkg/reconciler/channel/channel_test.go

```diff
@@ -1062,7 +1062,7 @@ func TestReconcileKind(t *testing.T) {
 			},
 		},
 		{
-			Name: "Reconciled normal - with single fresh subscriber - with auth - PlainText",
+			Name: "Reconciled normal - with single fresh subscriber - with auth - SSL",
```

Comment:
This test was using a SSL secret but the name contained PlainText, so renamed :)